lib/preds.nix

```diff
@@ -0,0 +1,17 @@
+{ lib }:
+with lib; {
```

Comment:
Document these individually, so `nixdoc` can pick them up properly.

Reply:
I think I will rename `preds.true` to `preds.any` and `preds.false` to `preds.none` to make the problematic example of @edolstra `["8.13" preds.true]` read `["8.13" preds.any]` instead, which looks more readable IMO.